connector/src/main/java/org/geysermc/connector/network/translators/item/ItemRegistry.java

```diff
@@ -150,6 +152,89 @@ public static void init() {
             }
         }
 
+        Object2IntMap<String> bedrockBlockIdOverrides = new Object2IntOpenHashMap<>();
+        Set<String> blacklistedIdentifiers = new ObjectOpenHashSet<>();
+
+        // Load creative items
+        // We load this before item mappings to get overridden block runtime ID mappings
+        stream = FileUtils.getResource("bedrock/creative_items.json");
+
+        JsonNode creativeItemEntries;
+        try {
+            creativeItemEntries = GeyserConnector.JSON_MAPPER.readTree(stream).get("items");
+        } catch (Exception e) {
+            throw new AssertionError(LanguageUtils.getLocaleStringLog("geyser.toolbox.fail.creative"), e);
+        }
+
+        int netId = 1;
+        List<ItemData> creativeItems = new ArrayList<>();
+        JsonNode damageNode;
+        JsonNode countNode;
+        JsonNode blockRuntimeIdNode;
+        JsonNode nbtNode;
+        for (JsonNode itemNode : creativeItemEntries) {
+            int count = 1;
+            int damage = 0;
+            int blockRuntimeId = 0;
+            NbtMap tag = null;
+            damageNode = itemNode.get("damage");
+            if (damageNode != null) {
+                damage = damageNode.asInt();
+            }
+            countNode = itemNode.get("count");
+            if (countNode != null) {
+                count = countNode.asInt();
+            }
+            blockRuntimeIdNode = itemNode.get("blockRuntimeId");
+            if (blockRuntimeIdNode != null) {
+                blockRuntimeId = blockRuntimeIdNode.asInt();
+            }
+            nbtNode = itemNode.get("nbt_b64");
+            if (nbtNode != null) {
+                byte[] bytes = Base64.getDecoder().decode(nbtNode.asText());
+                ByteArrayInputStream bais = new ByteArrayInputStream(bytes);
+                try {
+                    tag = (NbtMap) NbtUtils.createReaderLE(bais).readTag();
+                } catch (IOException e) {
+                    e.printStackTrace();
```

Comment:
Should some form of debug message be put here so if it happens we can work out why and where the stack trace came from?

Reply:
I feel like the stack trace pointing there would be detail enough? Either way, it can only happy in a development environment as that means the creative content JSON changed (so you have to deal with it then anyway).

Reply:
Ah yea good point.